src/ceph-volume/ceph_volume/devices/lvm/migrate.py

```diff
@@ -91,7 +91,7 @@ def ensure_associated_lvs(lvs, lv_tags):
                 device_uuid = lv.tags['ceph.{}_uuid'.format(type)]
             except KeyError:
                 # Bluestore will not have ceph.journal_uuid, and Filestore
-                # will not not have ceph.db_uuid
+                # will not have ceph.db_uuid
```

Comment:
this could be very wrong ...

I hope it isn't...

If this change is wrong, I can rewrite the text to not use a double negative